integrationm/common/com/cultofbits/customizations/audit/AuditUtils.groovy

```diff
@@ -0,0 +1,122 @@
+package com.cultofbits.customizations.audit
+
+import com.cultofbits.integrationm.service.actionpack.RecordmActionPack
+import com.cultofbits.integrationm.service.dictionary.recordm.Definition
+import com.cultofbits.integrationm.service.dictionary.recordm.FieldDefinition
+import com.cultofbits.integrationm.service.dictionary.recordm.RecordmMsg
+import org.json.JSONObject
+
+class AuditUtils {
```

Comment:
I think the name is not clear. Probably better `AuditInstanceFUpdater` and it should only have one public method: 

```
def static Map<String, String> getInstanceFieldsUpdated(Definition definition, RecordmMsg msg, UsermActionPack userm){}
```

Every other method should be private. Also could you place the public method as the first one in the class. This will make the code easier to read since it will be a top-down readig.